auto_backup/i18n/fa.po

```diff
@@ -0,0 +1,304 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * auto_backup
+#
+# Danaye N. <info@modoo.ir>, 2015.
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
```

Comment:
Did you re-generate this file for V15 @danayen? :)

Reply:
Yes, I just translated it based on English format.
Have I violated copyright? :D

Reply:
No absolutely no copyright violation or so and I'm very happy for your PR! 😄 
The following line of code:
```suggestion
"Project-Id-Version: Odoo Server 8.0\n"
```
Seems to suggest you didn't generate a new po file but just forwarded the 8.0?

Reply:
I copied a file from "i18n "folder then translated. and only wrote my name. I did not manipulate too much so that it does not break. :))
If there is other file, pls send me. I will do it.

Reply:
Can you take over the content from https://github.com/Yenthe666/auto_backup/blob/15.0/auto_backup/i18n/es.po and translate that into Persian? Then I am sure the `.po` file works fine and is 'migrated' to v15 :)

Reply:
Tranlated and replaced for V.15. 🤗